TF_Tooling_Web/src/TouchFree.ts

```diff
@@ -0,0 +1,29 @@
+import { ConnectionManager } from "./Connection/ConnectionManager";
+import { SVGCursor, SVGCursorParams } from "./Cursors/SvgCursor";
+import { TouchlessCursor } from "./Cursors/TouchlessCursor";
+import { WebInputController } from "./InputControllers/WebInputController";
+
+var InputController: WebInputController;
+export var CurrentCursor: TouchlessCursor;
+
+export interface TfInitParams {
+    initialiseCursor?: boolean;
+}
+
+export function Init(_tfInitParams?: TfInitParams ): void {
+    ConnectionManager.init();
+
+    ConnectionManager.AddConnectionListener(() => {
+        InputController = new WebInputController();
+
+        if (_tfInitParams === undefined) {
```

Comment:
IMO `if(!_tfInitParams)` looks cleaner but not a blocker

Reply:
`!var` can return true in a wide array of situations because of JS type enforcement (e.g. it'll also return true if it's `null` or if it's `false`) where checking explicitly only covers `undefined` which is what I want here

Reply:
This case it's only defined or undefined so a falsey check is fine imo (agreed if it you're checking something that can be true/false/undefined)